packages/flutter_tools/test/general.shard/run_hot_test.dart

```diff
@@ -29,6 +36,70 @@ void main() {
       const NoOpAnalytics(),
     );
   });
+
+  group('signal handling', () {
+    late final _FakeHotCompatibleFlutterDevice flutterDevice;
+
+    setUpAll(() {
+      flutterDevice = _FakeHotCompatibleFlutterDevice(FakeDevice());
+    });
```

Comment:
Nit: while I don't foresee this `group` growing to encompass a large number of tests, I prefer each test to do its own setup except in scenarios where setup is expensive or `testUsingContext` needs to be used.

https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#make-each-test-entirely-self-contained

Reply:
edit: I'm dumb, we are using `testUsingContext`